app/services/scheduler.py

```
@@ -307,6 +310,29 @@ def _add_disk_space_monitoring_job(self):
             logger.exception(f"Error adding disk space monitoring job: {e}")
 
 
+    def _add_storage_permissions_job(self):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_add_storage_permissions_job`

Add return type annotation: `None`

(ANN202)